slack-clone-compose-sample/src/main/kotlin/io/getstream/slack/compose/ui/channels/ChannelsScreen.kt

```diff
@@ -0,0 +1,194 @@
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.Spacer
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.size
+import androidx.compose.foundation.layout.width
+import androidx.compose.foundation.layout.wrapContentWidth
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.material.Card
+import androidx.compose.material.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.collectAsState
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.saveable.rememberSaveable
+import androidx.compose.runtime.setValue
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.clip
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.text.style.TextAlign
+import androidx.compose.ui.unit.dp
+import io.getstream.chat.android.client.models.Channel
+import io.getstream.chat.android.compose.ui.channel.header.ChannelListHeader
+import io.getstream.chat.android.compose.ui.channel.list.ChannelList
+import io.getstream.chat.android.compose.ui.common.SearchInput
+import io.getstream.chat.android.compose.ui.theme.ChatTheme
+import io.getstream.chat.android.compose.viewmodel.channel.ChannelListViewModel
+import io.getstream.slack.compose.R
+import io.getstream.slack.compose.model.Workspace
+import io.getstream.slack.compose.ui.util.isDirectGroupChat
+import io.getstream.slack.compose.ui.util.isDirectOneToOneChat
+
+@Composable
+fun ChannelsScreen(
+    listViewModel: ChannelListViewModel,
+    workspace: Workspace,
+    onItemClick: (Channel) -> Unit = {}
+) {
+    val currentUser by listViewModel.user.collectAsState()
+    val isNetworkAvailable by listViewModel.isOnline.collectAsState()
+
+    var searchQuery by rememberSaveable { mutableStateOf("") }
+
+    Column(
+        modifier = Modifier
+            .fillMaxSize()
+            .background(ChatTheme.colors.appBackground)
+    ) {
+
+        ChannelListHeader(
+            modifier = Modifier.height(56.dp),
+            currentUser = currentUser,
+            title = workspace.title,
+            isNetworkAvailable = isNetworkAvailable,
+            leadingContent = { WorkspaceLogo(logo = workspace.logo) },
+            titleContent = {
+                WorkspaceTitle(
+                    modifier = Modifier.weight(1f),
+                    title = workspace.title
+                )
+            },
+            trailingContent = { Spacer(Modifier.width(36.dp)) },
+        )
+
```

Comment:
Card just to render the shadow

Reply:
Can we use modifiers for this? And avoid having a card wrapper.

If not that's fine, but we may want to expose a way to have this customizable without using extra wrappers if possible